lib/scholar/naive_bayes/gaussian_naive_bayes.ex

```diff
@@ -0,0 +1,425 @@
+defmodule Scholar.NaiveBayes.GaussianNB do
```

Comment:
Do we need NB in the name given that we are already in the NaiveBayes namespace?

```suggestion
defmodule Scholar.NaiveBayes.Gaussian do
```

Reply:
Don't we have an imputer namespace, given the moduledoc

Reply:
Perhaps the moduledoc is leftover from copy-paste